cmd/occollector/app/collector/processors.go

```diff
@@ -93,21 +94,53 @@ func buildQueuedSpanProcessor(logger *zap.Logger, opts *builder.QueuedSpanProces
 			logger,
 			sender.HTTPTimeout(thriftHTTPSenderOpts.Timeout),
 		)
-	default:
-		logger.Fatal("Unrecognized sender type configured")
+	}
+
+	if spanSender == nil {
+		logger.Fatal("Unrecognized sender type or no exporters configured")
```

Comment:
It would be useful for this Fatal to be a Fatalf and include the opts.SenderType.